Specs/Number/Chinese/NumberRangeModel.json

```diff
@@ -0,0 +1,422 @@
+[{
+  "Input": "这个数字大于二十的同时小于等于三十五",
+  "NotSupported": "javascript",
+  "NotSupportedByDesign": "python",
+  "Results": [
+    {
+      "Text": "大于二十的同时小于等于三十五",
+      "TypeName": "numberrange",
+      "Resolution": {
```

Comment:
The schema should be similar to datetime ranges (ex: DateTimePeriodParser.json)

Reply:
Inside "Value" there could be one field called "Resolution"
that filed could follow a standard similar to Yiyong's. Or have two attributes.

Reply:
Let's keep "value" as a string. And use Yiyong's representation for intervals. Open ended will be empty. ex: >0 is (0,)
= 0 is [0,0]

Reply:
Changed as we discussed.